sdk/cognitivelanguage/Azure.AI.Language.Conversations/tsp-location.yaml

```diff
@@ -0,0 +1,5 @@
+directory: specification/cognitiveservices/Language.Conversations
+additionalDirectories: []
+commit: e57839dc71bb80d16a8038eb3829ed4d4b2cc466
+repo: Zach-King2/azure-rest-api-specs
```

Comment:
This should be Azure/azure-rest-api-specs and use the current HEAD commit of your feature branch.

Reply:
In order to make this change, I first have to merge the changes I made to the tspconfig.yaml file in order for the SDK generation to work. I'll create a PR for that now.